product_docs/docs/pem/8/registering_agent.mdx

```diff
@@ -58,6 +58,58 @@ Before any changes are made on the PEM database, the connecting agent is authent
 
 If you don't provide the password, a password authentication error occurs. You are prompted for any other missing required information. When the registration is complete, the server confirms that the agent was successfully registered.
 
+Each registered PEM agent must have a unique agent ID. For 8.7 and later, 
+
+-   The value `max(id)+1` is assigned to each agent ID. 
+-   You can override the agent ID manually using the command line.
+
+!!! Note
+    For 8.6 and earlier, the sequence `pem.agent_id_seq` generates and gives a unique ID to the PEM agent. 
+
+### Examples
+
+This example shows how to register the PEM agent overriding the default configurations using the `-o` option for 8.7 and later.
+
+Register the PEM agent using command line. Assign agent ID 8 to override the default.
+
+```shell{promptUser: root}{promptHost: pem}{outputLines: 2-8}
+/usr/edb/pem/agent/bin/pemworker --register-agent \
+--pem-server pemserver \
+--pem-user postgres \
+--pem-port 5432 \
+--display-name agent8 \
+-o agent_id=8
+__OUTPUT__
+Postgres Enterprise Manager Agent registered successfully!
+```
+
+It successfully regsiters the PEM agent because the ID 8 is available. If the given ID is already in use by the existing agent, it throws an error.
```

Comment:
typo (registers). Also, I would put the explanation first: Because the agent_id of 8 is available, the agent registers successfully.

I don't think you need to mention the error condition at all.